_includes/manuals/3.14/1.2_release_notes.md

```diff
@@ -4,6 +4,17 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### IPV6 Support
+Satellite deployment and operation in IPv6 networks is now fully supported. Dual stack deployments have not been tested and are not supported.
```

Comment:
Please fix the branding. In upstream we also have a different definition of supported. Red Hat interprets supported as you can call support if it's broken while in upstream you can only ask around in the community. They don't care about supported vs not supported so probably leave it out and emphasize IPv6-only.

Something like this perhaps?

```suggestion
Deployment and operation in IPv6-only networks is now fully supported.
```